playwright/integrations/unauthenticated-chat.spec.ts

```diff
@@ -366,8 +366,8 @@ test.describe('UnauthenticatedChat @UnauthenticatedChat', () => {
         const {acsEndpoint, chatId} = runtimeContext;
         const getMessagesRequestPartialUrl = `${acsEndpoint}chat/threads/${encodeURIComponent(chatId)}/messages?api-version=`;
         const getMessagesResponseDataJson = await getMessagesResponse.json();
-        const sentMessages = getMessagesResponseDataJson.value.filter((message) => message.type === 'text');
-        const sentMessageContent = sentMessages[0].content.message;
+        const sentMessage = getMessagesResponseDataJson.value.filter((message) => message.type === 'text' && message.content.message === content);
+        const sentMessageContent = sentMessage[0].content.message;

```

Comment:
should we check for null?

Reply:
We shouldn't. It's an integration test to validate `ChatSDK.getMessages()`. In this test, we're sending a message first, then should expect the message that was sent to be part of the list of messages.